documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1,4 +1,6 @@
-= Debezium Connector for MongoDB
+= {ProductName} Connector for MongoDB
```

Comment:
This substitution fails on the community docs build because we source substitutions from `_attributes.adoc` which is actually included below, so Antora renders this as `{ProductName}`.  In the MySQL PR we had other substitution issues because the modularized documentation didn't include the `_attributes.adoc`.  I propose we use `{prodname}` which will render on the community side since we defined that in the playbook when merging the MySQL work.